### DISCUSSION TITLE
istore os使用Tailscale进行远程访问istore后台出现304错误。

### DISCUSSION THREAD

**Discussion by ypmysjy** [istoreos/istoreos#1538] [Q&A]
istore os使用Tailscale进行远程访问istore后台出现304错误，其他端口正常，包括在同局域网的梅林ax68U，老毛桃斐讯k2都可以访问后台。我想知道是不是我的istore设置错误了，或者你们有相同的情况，可以指点一下。

**Comment by jjm2473** [comment]
304是错误吗？https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Status/304
